src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -41,34 +41,94 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({ author }) => {
     socialNetwork: author
       ? author.socialNetworks
       : [null, null, null, null, null],
-    work: author ? author.mainInstitution.name : null,
+    work: author ? author.mainInstitution.name : '',
   });
-  const [errorFields, setErrorFields] = useState<IErrorFields>({
-    avatar: author?.avatar ? '' : i18n.t(langTokens.admin.pictureRequired),
-    lastName: author?.lastName ? '' : i18n.t(langTokens.admin.fieldCantBeEmpty),
-    firstName: author?.firstName
-      ? ''
-      : i18n.t(langTokens.admin.fieldCantBeEmpty),
-    regionId: author?.region.id
-      ? ''
-      : i18n.t(langTokens.admin.fieldCantBeEmpty),
-    cityId: author?.mainInstitution.city.id
+
+  const errorMessages = useMemo(() => {
+    const errors: IErrorFields = {
+      avatar: author?.avatar ? '' : i18n.t(langTokens.admin.pictureRequired),
```

Comment:
1. here we are checking fields from `author`, later we check fields from `newAuthorValues`, please explain, I'm confused why we are validating the same fields from props and from state
2. even if we need to validate from both objects, we should add `author` to dependency array of `useMemo`. I'll raise this, we have `react-hooks/exhaustive-deps` disabled in our ESLint config and that's pretty dangerous.
3. whether or not we want to validate both `author` and `newAuthorValues` I believe it's better to change pattern for validation. Current:
```
errors = {
  email: ...
  ....
};
error.email = validateInput(...)
```
just overwrites messages from first block with result of `validateInput` call. It might be not a defect if you get desired error message finally, but it's really, really confusing.
So rule of thumb: setting single error key just once:
```
const error = {
...
  email: '',
...
};
if (...) {
  error.email = ...
} else if {
  error.email = ....
}
```